### ISSUE TITLE
[Bug]: Alignment of field labels are off on generate CRUD page

### ISSUE THREAD

**Issue by ramsaptami** [appsmithorg/appsmith#10194]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Field labels are not uniformly aligned on the generate CRUD page.

![image](https://user-images.githubusercontent.com/79509062/148335476-93eab57f-ab3b-403b-8447-9d1f2692bb88.png)

### Steps To Reproduce

1. Add a new page and select CRUD option and observe the label alignment

### Environment

Release

### Version

Cloud

**Comment by Rishabh-Rathod** [comment]
@jsartisan  Can you check if this was due to tailwind changes or not?

**Comment by jsartisan** [comment]
Let me check

**Comment by jsartisan** [comment]
@Rishabh-Rathod Yes. This was due to tailwind only. I'll fix this. Let me know if there are any other places where things might be misaligned due to tailwind.

**Comment by Rishabh-Rathod** [comment]
@jsartisan can you search for "ADS component issue" labelled issues. I created that particular label for such internal issues to help me find them.

**Comment by RakshaKShetty** [comment]
Still see the same issue so reopening the issue 
https://github.com/appsmithorg/appsmith/issues/8438

**Comment by Rishabh-Rathod** [comment]
@RakshaKShetty this seems strange as for me.

On DP
<img width="682" alt="Screenshot 2022-01-11 at 8 02 29 PM" src="https://user-images.githubusercontent.com/23132741/148962353-2f5ddd2c-9715-4789-a7c6-0b7bce46d5b8.png">


On Release
<img width="1272" alt="Screenshot 2022-01-11 at 8 03 10 PM" src="https://user-images.githubusercontent.com/23132741/148962398-66eac0a3-35a8-4c8d-b0d1-018a7bd884a0.png">

This did work as expected. Can you please check again?

**Comment by ramsaptami** [comment]
The alignment is as expected on release now. Thanks @Rishabh-Rathod
